airflow_healthcheck.py

```diff
@@ -0,0 +1,18 @@
+import json
+import sys
+from urllib.request import urlopen
+
+HEALTHY = 'healthy'
+url = 'http://airflow_webserver:8080/health'
```

Comment:
hardcoded relative URL in image - won't work outside of this `docker-compose` configuration; I suggest to use `localhost` and only have the healthcheck on `webserver`, not putting one on `scheduler` for now

Reply:
updated.